src/main.tsx

```
@@ -3,6 +3,7 @@ import { createRoot } from "react-dom/client";
 import { BrowserRouter } from "react-router-dom";
 import "./index.css";
 
+import { Toaster } from "react-hot-toast";
```

Biome (2.1.2)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Unsafe fix: Remove the unused imports.


(lint/correctness/noUnusedImports)